misaligned.js

```diff
@@ -3,12 +3,28 @@ const {expect} = require('chai')
 function print_color_map() {
     const majorColors = ["White", "Red", "Black", "Yellow", "Violet"];
     const minorColors = ["Blue", "Orange", "Green", "Brown", "Slate"];
-    for (let i = 0; i < majorColors.length; i++) {
-        for (let j = 0; j < minorColors.length; j++) {
-            console.log(`${i * 5 + j} | ${majorColors[i]} | ${minorColors[j]}`);
+    let i = 0;
+    let j = 0;
+    let longestStringLength = getLongestString(majorColors);
+    for (i = 0; i < majorColors.length; i++) {
+        for (j = 0; j < minorColors.length; j++) {
+            if (i * 5 + j+1 < 10) {
+                console.log(`${i * 5 + j+1} | ${majorColors[i].padEnd(longestStringLength+1)}| ${minorColors[j]}`);
+            } else {
+                console.log(`${i * 5 + j+1}| ${majorColors[i].padEnd(longestStringLength)} | ${minorColors[j]}`);
+            } 
         }
     }
-    return majorColors.length * minorColors.length;
+    return i * j;
+}
+function getLongestString(majorColors) {
+    let longestString = "";
+    for (i = 0; i < majorColors.length; i++) {
+        if (longestString.length <= majorColors[i].length) {
+            longestString = majorColors[i];
+        }
+    }
+    return longestString.length;
 }
 
 result = print_color_map();

```

Comment:
how is the content of the color map verified? there should be asserts for that too.